lib/tasks/remove_location_cateogory_from_subscription_search_criteria.rake

```diff
@@ -2,8 +2,10 @@ namespace :subscription do
   desc "remove location_category from search_criteria"
   task remove_location_category_from_search_criteria: :environment do
     Subscription.find_each do |subscription|
-      subscription.search_criteria["location"] = subscription.search_criteria.delete("location_category")
-      subscription.save
+      if subscription.search_criteria["location_category"].present? && subscription.search_criteria["location"].blank?
+        subscription.search_criteria["location"] = subscription.search_criteria.delete("location_category")
```

Comment:
Would it be OK to not delete the `location_category` for now (would that break anything)? I prefer idempotent tasks where possible in case they break half way through. https://stackoverflow.com/questions/1077412/what-is-an-idempotent-operation#:~:text=at%204%3A12-,12,as%20making%20a%20single%20request.